erpnext/accounts/report/trial_balance/trial_balance.py

```diff
@@ -87,8 +87,7 @@ def get_opening_balances(filters):
 
 
 def get_rootwise_opening_balances(filters, report_type):
-	additional_conditions = " and posting_date >= %(year_start_date)s" \
-		if report_type == "Profit and Loss" else ""
+	additional_conditions = ""
```

Comment:
Accountants in India don't expect this value in Trial Balance. Thats why I made it optional by adding a filter. Added this commit:
https://github.com/frappe/erpnext/commit/437b32d32c524efdc4930068204f696ca29e00fe